signer/src/config/default.toml

```diff
@@ -40,14 +40,22 @@ endpoints = ["http://localhost:20443"]
 nakamoto_start_height = 31
 
 # !! ==============================================================================
-# !! Signer Stacks Account Configuration
+# !! Signer Configuration
 # !! ==============================================================================
-[signer.stacks_account]
-# The private key associated with the provided Stacks address.
+[signer]
+# The private key associated with the signer. This is used to generate the
+# signers associated public key and sign messages to other signers.
 #
 # Required: true
 private_key = "8183dc385a7a1fc8353b9e781ee0859a71e57abea478a5bca679334094f7adb5"
-
+# Specifies which network to use when constructing and sending transactions
+# on stacks and bitcoin. This cooresponds to the `chain` flag in the
+# bitcoin.conf file of the connected bitcoin-core node, and the
+# `burnchain.mode` flag int he config.toml of the connected stacks-core
+# node.
+#
+# Required: true
```

Comment:
I'd add:
```
# Possible Values: mainnet, testnet, regtest
# Environment: SIGNER_SIGNER__NETWORK
```

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/474/commits/57efea14a608dde5e06ab2393a77221ae3b2dde4